cfme/scripting/tests/test_quickstart.py

```diff
@@ -46,10 +47,8 @@ def yamls_volume():
 
 
 @pytest.mark.parametrize('image, prepare', IMAGE_SPEC, ids=[x[0] for x in IMAGE_SPEC])
-@pytest.mark.parametrize('python', [
-    'python2',
-    pytest.param('python3', marks=pytest.mark.xfail(reason="dump2polarion bug"))
-])
+@pytest.mark.parametrize('python',
+                         [pytest.param('python3.7')])
```

Comment:
Q: why not `python3`?

Reply:
Being explicit in test parametrization about what python version is tested. Assuming that we will at some point have 3.8, 3.9, 4.0, etc here as well.